docs/se/guides/health.adoc

```diff
@@ -158,96 +165,129 @@ curl http://localhost:8080/observe/health
     }
   ]
 }
-
 ----
+<1> Overall application health status
+<2> List of individual health checks.
 
+=== Adding Custom Health Checks
 
+You can add your own custom health checks. These typically assess the conditions in and around your application and report whether the service should be considered started, live, and/or ready.
 
-=== Adding Custom Health Checks
+The following trivial but illustrative example adds a custom start-up check that reports `DOWN` until the server has been running for eight seconds and reports `UP` thereafter. Note the two main steps in the example code:
 
-As illustrated above, the generated `Main` class in the QuickStart project adds two custom health checks.
-Your own checks typically assess the conditions in and around your application and report whether the service should be considered started, live, and/or ready.
+1. Create an explicit instance of `ObserveFeature` which contains a custom `HealthObserver` with the custom check.
+2. Add that `ObserveFeature` instance to the `WebServer.Builder` as a feature.
 
-As a trivial but illustrative example, the custom start-up health check in the generated `Main` class reports that the app is _not_ started until eight seconds after the server has started. After that, the start-up check reports that the app _is_ started.
 
-To see this start-up check in action, stop your running server (press ^C), then rerun it.
+[source,java]
+.Updated `Main.main`, augmenting the creation of `WebServer` instance with a custom health check
+----
+AtomicLong serverStartTime = new AtomicLong();  // <1>
+
+ObserveFeature observe = ObserveFeature.builder()
+        .config(config.get("server.features.observe")) // <2>
+        .addObserver(HealthObserver.builder() // <3>
+                             .details(true) // <4>
+                             .addCheck(() -> HealthCheckResponse.builder() // <5>
+                                               .status(System.currentTimeMillis() - serverStartTime.get() >= 8000)
+                                               .detail("time", System.currentTimeMillis())
+                                               .build(),
+                                       HealthCheckType.STARTUP,
+                                       "warmedUp")
+                             .build()) // <6>
+        .build();
+
+    WebServer server = WebServer.builder()
```

Comment:
Extra indentation here, right?

Reply:
I retyped the source and our formatting template did trim some of the indentation. Fixed.